pkg/operator/ceph/cluster/crash/add.go

```diff
@@ -39,7 +39,8 @@ import (
 const (
 	controllerName = "ceph-crashcollector-controller"
 	// AppName is the value to the "app" label for the ceph-crash pods
-	AppName = "rook-ceph-crashcollector"
+	AppName    = "rook-ceph-crashcollector"
+	PrunerName = "rook-ceph-crashcollector-pruner"
```

Comment:
Why is `PrunerName` exported?

Reply:
No reason, just a habit of capitalizing global vars will change.